Qualitätsanforderungen.md

```diff
@@ -0,0 +1,13 @@
+## Qualitätsanforderungen
+
+* Variablendeklarationen werden in Englisch verfasst.
+* Alle Kommentare müssen den Javadoc-Kommentarandorderungen genügen.
+* Funktionen werden mit zwei Tabstops, der Rest des Quellcodes mit einem Tabstop eingerückt.
+* In Funktionen werden Rückgabewerte immer geprüft, bevor ein Wert zurück gegeben wird.
+* Eingaben werden immer überprüft und mögliche Fehler durch Exceptions abgefangen.
+* Der Quellcode folgt dem Prinzip KISS (Keep it stupid simple) und DRY (Don't repeat yourself).
+* Das Umbrechen einer Quellcodezeile ist zu vermeiden.
+* Die Implementierung muss jederzeit durch andere Projektmitglieder verstanden werden können.
+* Jegliche Änderungen am Quellcode werden in einem extra Branch verwaltet und nur über einen Pull-Request in den master-Branch gemerged. Jeder Pull-Request wird von Devwurm, reviloves und denisk176 kontrolliert. 
```

Comment:
Ich würde die Reviewer nicht festlegen. Ich denke es sollten 2 Reviewer benannt werden, die selbst nicht an der Implementierung mitgewirkt haben und diese müssen beide zustimmen

Reply:
Wen schlägst du hier vor ? Es sollen ja alle an der Implementierung arbeiten ...

Reply:
"Jegliche Änderungen am Quellcode werden in einem extra Branch verwaltet und nur über einen Pull-Request in den master-Branch gemerged. Jeder Pull-Request wird von 2 Teammitgliedern, die selbst nicht an der Implementierung dieses Branches beteiligt waren, reviewed. "
Oder so